nvcf/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG BASE_IMAGE
+ARG BASE_TAG
+FROM ${BASE_IMAGE}:${BASE_TAG}
+
+ARG NGC_KEY
+ENV NGC_API_KEY=${NGC_KEY}
+USER root
```

Comment:
Do we really require NIMs to run as root in NVCF?

Reply:
This can be removed